std/php7/ArrayAccess.hx

```diff
@@ -0,0 +1,13 @@
+package php7;
+
+/**
+	Native PHP interface.
+	@see http://php.net/manual/en/class.arrayaccess.php
+**/
+@:native('ArrayAccess')
+extern interface ArrayAccess<K,V> {
```

Comment:
Would it be possible to make this work in a native way? So that implementing this on a Haxe class adds Haxe array access as well?